app/routes/photos/components/GalleryDetail.js

```diff
@@ -22,7 +23,8 @@ type Props = {
   fetch: (galleryId: Number, args: { next: boolean }) => Promise<*>,
   push: string => Promise<*>,
   uploadAndCreateGalleryPicture: (ID, File | Array<DropFile>) => Promise<*>,
-  actionGrant: Array<string>
+  actionGrant: Array<string>,
```

Comment:
There's a separate `ActionGrant` type in `apps/models`, in case you didn't know. 100% doesn't matter, though.

Reply:
:+1: Yeah, that sounds like a good idea.